ccc/parameters.py

```diff
@@ -230,9 +226,9 @@ def update_specification(self, revision, raise_errors=True):
                 `PARAM: YEAR-VALUE-DICTIONARY` pairs
 
             raise_errors (boolean):
-                if True (the default), raises ValueError when `parameter_errors`
+                if True (the default), raises ValueError when `_errors`
                     exists; if False, does not raise ValueError when
-                    `parameter_errors` exists and leaves error handling
+                    `_errors` exists and leaves error handling
```

Comment:
@jdebacker You can check the `errors` property instead of the private `_errors` attribute.

Reply:
Good suggestion - I've made this change.